frontend/src/components/question-creation/question-elements/SaveQuestionEditorButton.tsx

```diff
@@ -87,7 +92,13 @@ const SaveQuestionEditorButton = ({
 
   const validateNoEmptyElementErrors = () => {
     const emptyElement = questionElements.find(
-      (element) => element.data === "",
+      (element) =>
```

Comment:
this validation check wasn't working for question text and texts earlier
- not sure if code was changed from your initial PR